arch/arm64/configs/bcm2711_defconfig

```diff
@@ -784,6 +784,7 @@ CONFIG_VIDEO_TVP5150=m
 CONFIG_VIDEO_TW2804=m
 CONFIG_VIDEO_TW9903=m
 CONFIG_VIDEO_TW9906=m
+CONFIG_VIDEO_IMX219=m
```

Comment:
FYI, regenerating the defconfigs loses this setting.

Reply:
Thanks for the heads up.
I think we're missing CONFIG_VIDEO_V4L2_SUBDEV_API, which will drop out almost all of the sensor drivers :-(

Reply:
Suffice to say, I haven't been testing on 64bit kernels as yet.

Reply:
Looking at it, lots of options are missing from arm64/bcm2711_defconfig.
Time for a resync between arm and arm64.

Reply:
9532eb3c84d8d952ef28da3d135683ac01adc9b8 fixes the immediate problem.

Reply:
Thanks, but as CONFIG_VIDEO_BCM2835_UNICAM is also missing you get a sensor driver without anything to connect it to.

All the NFTABLES stuff, CGROUP_PIDS, I2C_MUX, LIRC/RC_CORE, USB DVB tuners, some BCM2708_SOC_DACs, and some other stuff is missing. I've just through with Meld and copied across all the obvious stuff. Now seeing if it builds.....

I'll throw it up as a PR so you can see what it proposed. There are still a few I haven't pulled over as they look processor related so I'm unsure if they are still relevant.

Reply:
> Thanks, but as CONFIG_VIDEO_BCM2835_UNICAM is also missing you get a sensor driver without anything to connect it to.

That's a secondary issue as far as I'm concerned. I just didn't want the IMX219 config setting to get lost as a side effect of adding some new ones.